images/twitter.html

```diff
@@ -0,0 +1,308 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Twitter Challenge</title>
+    <link rel="stylesheet" href="style.css">
+    </head>
+
+<body class="body">
+
+    <!--Left Menu Bar-->
+<div class="left-menu-bar-section">
+    <div class="left-menu-header">
+        <a class="logo"><img src="./Assets/the-avengers-svgrepo-com.svg"></a> 
+    </div>
+
+    <!--Nav Bar-->
+
+    <div class="nav">
```

Comment:
Not sure why but the navbar images won't load for me, not quite sure what's going on there since the rest load just fine.